markdown/1.2/en/ff-middleware.md

```diff
@@ -125,3 +125,27 @@ window.addEventListener("ffReady", () => {
 ```
 
 See the [API tab](/api/1.2/ff-middleware#tab=api) for more details on configuration options.
+
+#### ProductTeaserCampaignProcessor
+
+Starting with FACT-Finder version 7.3 and FACT-Finder Web Components version 1.2.16 _feedback text campaigns_ can be configured as _teaser_.
```

Comment:
I wonder if we should be more precise about the version. A _7.3.5-42_ is needed

Reply:
If there is no doubt it is indeed that version, I'll add it quickly.